fastplotlib/layouts/_plot_area.py

```diff
@@ -117,6 +117,9 @@ def __init__(
         self._background = pygfx.Background(None, self._background_material)
         self.scene.add(self._background)
 
+        self.scene.add(pygfx.AmbientLight())
+        self.scene.add(self._camera.add(pygfx.DirectionalLight()))
```

Comment:
I think these can be settable properties of the `PlotArea`

Reply:
I think it's useful to have easy access to them though, and if a new camera is set on the `PlotArea` it needs to get the directional light. This is what I mean:

https://github.com/fastplotlib/fastplotlib/pull/953/files#diff-2a0f782c81512ab17081f35fd6cac201fafd429eded696cb76981946bc6baaecR120-R294